jina/resources/executors._merge_chunks.yml

```diff
@@ -0,0 +1,10 @@
+!BaseExecutor
+with: {}
+metas:
+  name: merge
+requests:
+  on:
+    [SearchRequest, TrainRequest, IndexRequest]:
+      - !ReduceAllDriver {}
```

Comment:
For the sake of correctness, this should be
```
- !ReduceAllDriver
  with:
    traversal_paths: ['c']
```
and not `['c', 'r']`.

Reply:
it is only c by definition of ReduceAllDriver